examples/data-objects/monaco/src/index.ts

```diff
@@ -23,14 +22,6 @@ const componentFactory = new DataObjectFactory(
     {},
 );
 
-const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore(
-    componentFactory,
-    new Map([
-        [monacoName, Promise.resolve(componentFactory)],
-    ]),
-);
+const monacoViewCallback = (model: MonacoRunner) => new MonacoRunnerView(model.text);
 
-export const fluidExport: IProvideFluidDataStoreFactory & IProvideRuntimeFactory = {
-    IFluidDataStoreFactory: componentFactory,
-    IRuntimeFactory: runtimeFactory,
-};
+export const fluidExport = new ContainerViewRuntimeFactory(componentFactory, monacoViewCallback);
```

Comment:
Ah, this looks like another place expecting the `request` pattern to pass the data.

Reply:
Internally the ContainerViewRuntimeFactory requests the default component (basically the same as what ContainerRuntimeFactoryWithDefaultDataStore does).